constants/petnums.json

```diff
@@ -3907,21 +3903,17 @@
   "RIFT_FERRET": {
     "EPIC": {
       "1": {
-        "otherNums": [
-          10
-        ],
+        "otherNums": [],
         "statNums": {
           "SPEED": 0.5,
-          "INTELLIGENCE": 0.02
+          "INTELLIGENCE": -0.02
```

Comment:
same here.

Reply:
![{ED27BD31-185A-478A-B98C-55DCE1D26804}](https://github.com/user-attachments/assets/57cd9fc0-2d99-41dd-a893-244b6560f7c8)
should i change the jerry pet too? i didn't do this it was already like that so i just copied the only other pet with negitive stats

Reply:
Oh i didnt know we already had jerry with that shit, ig its fine